list1.h

```diff
@@ -0,0 +1,22 @@
+#ifndef LIST1_H
```

Comment:
should include a copyright and a class description.  i know this is a work in progress, but i think it's easiest to get those type of standard thing out of the way as early as possible, so as not to forget them :)